packages/office-ui-fabric-react/src/components/Slider/Slider.tsx

```diff
@@ -175,31 +183,56 @@ export class Slider extends BaseComponent<ISliderProps, ISliderState> implements
     let currentSteps: number | undefined;
-
-    if (event.type === 'mousedown' || event.type === 'mousemove') {
-      currentSteps = getRTL() ?
-        (sliderPositionRect.right - (event as MouseEvent).clientX) / stepLength :
-        ((event as MouseEvent).clientX - sliderPositionRect.left) / stepLength;
-    } else if (event.type === 'touchstart' || event.type === 'touchmove') {
-      currentSteps = getRTL() ?
-        (sliderPositionRect.right - (event as TouchEvent).touches[0].clientX) / stepLength :
-        ((event as TouchEvent).touches[0].clientX - sliderPositionRect.left) / stepLength;
+    if (!this.props.vertical) {
+      if (event.type === 'mousedown' || event.type === 'mousemove') {
+        currentSteps = getRTL() ?
+          (sliderPositionRect.right - (event as MouseEvent).clientX) / stepLength :
+          ((event as MouseEvent).clientX - sliderPositionRect.left) / stepLength;
+      } else if (event.type === 'touchstart' || event.type === 'touchmove') {
+        currentSteps = getRTL() ?
+          (sliderPositionRect.right - (event as TouchEvent).touches[0].clientX) / stepLength :
+          ((event as TouchEvent).touches[0].clientX - sliderPositionRect.left) / stepLength;
+      }
+    } else {
+      if (event.type === 'mousedown' || event.type === 'mousemove') {
+        currentSteps = getRTL() ?
+          (sliderPositionRect.bottom - (event as MouseEvent).clientY) / stepLength :
+          ((event as MouseEvent).clientY - sliderPositionRect.top) / stepLength;
+      } else if (event.type === 'touchstart' || event.type === 'touchmove') {
+        currentSteps = getRTL() ?
+          (sliderPositionRect.bottom - (event as TouchEvent).touches[0].clientY) / stepLength :
+          ((event as TouchEvent).touches[0].clientY - sliderPositionRect.top) / stepLength;
+      }
     }
 
     let currentValue: number | undefined;
     let renderedValue: number | undefined;
 
     // The value shouldn't be bigger than max or be smaller than min.
     if (currentSteps! > Math.floor(steps)) {
-      renderedValue = currentValue = max as number;
+      if (!this.props.vertical) {
+        renderedValue = currentValue = max as number;
+      } else {
+        renderedValue = currentValue = min as number;
+      }
     } else if (currentSteps! < 0) {
-      renderedValue = currentValue = min as number;
+      if (!this.props.vertical) {
+        renderedValue = currentValue = min as number;
+      } else {
+        renderedValue = currentValue = max as number;
+      }
     } else {
-      renderedValue = min! + step! * currentSteps!;
-      currentValue = min! + step! * Math.round(currentSteps!);
+      if (!this.props.vertical) {
+        renderedValue = min! + step! * currentSteps!;
+        currentValue = min! + step! * Math.round(currentSteps!);
+      } else {
+        renderedValue = max! - step! * currentSteps!;
+        currentValue = max! - step! * Math.round(currentSteps!);
```

Comment:
instead of duplicating all this code, maybe you can just flip min/max for vertical? This reduces bundle size and code that needs to be maintained.